File: flask-api/api/models/Offices.py
from .. import db

class Office(db.Model):
    """Model for offices. """
    
    __tablename__ = 'Office'

    id = db.Column(db.Integer, primary_key=True)
    name = db.Column(db.String(128), index=True)
    phone_number = db.Column(db.String(16), index=True)
    address = db.Column(db.String(128), index=True)
    city = db.Column(db.String(32), index=True)
    state = db.Column(db.String(16), index=True)
    provider_id = db.Column(db.Integer, db.ForeignKey('Provider.id'), nullable=False)

    
    
    

File: react-ui/src/components/OfficeComponents/Offices.js
import React from 'react';
import OfficeDataService from '../../services/office.service'
import Table from 'react-bootstrap/Table'

export default function App() {
  const [offices, setOffices] = React.useState(null);

  React.useEffect(() => {
    OfficeDataService.getAll().then((response) => {
      setOffices(response.data);
    });
  }, []);

  if (!offices) return null;

  return (
    <div>
        <Table striped bordered hover>
            <tr>
                <th>Office Name</th>
                <th>Address</th>
                <th>City</th>
                <th>State</th>
                <th>Phone Number</th>
                <th>Physicians</th>
            </tr>
            <tbody>
            {offices.map((office) => (
            <tr key={office.id}>
                <td>{office.name}</td>
                <td>{office.address}</td>
                <td>{office.city}</td>
                <td>{office.state}</td>
                <td>{office.phone_number}</td>
                <td></td>
            </tr>
                ))}
            </tbody>
      </Table>
      
    </div>
  );
}